tests/migration_test.go

```diff
@@ -725,8 +721,11 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				pvName := "test-iscsi-dv" + rand.String(48)
 				// Start a ISCSI POD and service
 				By("Starting an iSCSI POD")
-				iscsiIP := tests.CreateISCSITargetPOD(cd.ContainerDiskEmpty)
-				_, err = virtClient.CoreV1().PersistentVolumes().Create(tests.NewISCSIPV(pvName, "2Gi", iscsiIP, k8sv1.ReadWriteMany, k8sv1.PersistentVolumeFilesystem))
+				iscsiTargetPod := tests.CreateISCSITargetPOD(cd.ContainerDiskEmpty)
+				iscsiTargetIPAddress := libnet.GetPodIpByFamily(iscsiTargetPod, k8sv1.IPv4Protocol)
```

Comment:
nit: 
could be in one line: 
`Expect(libnet.GetPodIpByFamily(iscsiTargetPod, k8sv1.IPv4Protocol)).NotTo(BeEmpty())`

Reply:
It is necessary to pass `iscsiTargetIPAddress` to `NewISCSIPV` (line 728)

Reply:
Right, sorry,  my bad